main.md

```diff
@@ -679,16 +679,137 @@ TBD
-This is because SD-JWT already reveals information about the issuer and the schema,
-and revealing the claim names does not provide any additional information.
+	1.	Consent and choice 
+	2.	Purpose legitimacy and specification 
+	3.	Collection limitation 
+	4.	Data minimization 
+	5.	Use, retention and disclosure limitation 
+	6.	Accuracy and quality 
+	7.	Openness, transparency and notice 
+	8.	Individual participation and access 
+	9.	Accountability 
+	10.	Information security 
+	11.	Privacy compliance 
 
-## Unlinkability 
+Among them, this document is concerned directly with 3. Collection limitation and indirectly/partially with 1. Consent and choice, 2. Purpose legitimacy and specification, 6. Accuracy and quality, and 10. Information security.  
 
-It is also important to note that this format enables selective disclosure of claims, but
-in itself it does not achieve unlinkability of the subject of an SD-SWT.
+Often talked unlinkability is a subset of 3. Collection limitation as it amounts to providing unintended (by the holder) information such as the fact that the holder visited a verifier to another verifier. 
+
+## Collection limitation
+
+In this document, Collection limitation is considered from two aspects. 
+
+1. Uncoverability
+2. Unlinkability
+
+### Uncoverability
+
+The concept of uncoverability was discussed in clause 3. In this subclause, this document further discusses it in more detail. 
+
+#### Uncovering the values of non-existent claims from issuer identifier
+When an issuer issues only one type of SD-JWT, it is often possible to recover some attributes about the holder. 
+For example, if the issuer was an authority that only issues driver's licenses in this format, 
+then, just by looking at the issuer and the document itself, it can be deterministically recovered that the holder has a driving license. While revealing that the holder has a driving license may not have much privacy impact, 
+if it was other attributes, it could be quite a privacy-sensitive. 
+
+For example, suppose the case where the SD-JWT was issued by National Cancer Centre. 
+Then, by just looking at the issuer, it is possible to deduce that the holder is a cancer patient. 
+This may have a fair amount of privacy impact. 
+
+To mitigate this kind of issue, a group of issuers may elect to use a common issuer identifier and use a group signature scheme instead of an individual signature. 
+
+#### Uncovering the values of non-existent claims from the list of claim names
+Even if a group of issuers uses a common issuer identifier, if the claims used are unique from one issuer to another, 
+the list of the claim names would be enough to identify the real issuer. In this case, the values of non-existent claims can be uncovered in the same manner as described in the previous subclause. 
+
+To mitigate this kind of issue, the group of issuers may elect to use a common set of claims and always include them in the SD-JWT. 
```

Comment:
I think it is worth discussing this approach also independent of the problem of identifying the issuer. Always including some claims prevents verifiers from learning something about the holder just from the presence of a claim.

Example: In EU driver's licenses, there may be restrictions such as "may only drive with hearing aids" or "may only drive with modified vehicles to accommodate a certain disability". The verifier may learn just from the presence of any restrictions that the owner of the license may not be of best health, potentially using it to their disadvantage.

Always including claims and just nulling the value when not applicable can prevent this.

Reply:
The following proposal resolve by design the problem of the claim names disclosed in SD-JWT
https://github.com/oauthstuff/draft-selective-disclosure-jwt/issues/80